firebase-web/src/main/java/io/spine/web/firebase/FirebaseQueryProcessingResult.java

```diff
@@ -35,12 +37,15 @@
  */
 final class FirebaseQueryProcessingResult implements QueryProcessingResult {
 
-    private static final String MIME_TYPE = "text/plain";
+    @SuppressWarnings("DuplicateStringLiteralInspection") // The duplication is a coincidence.
+    private static final String MIME_TYPE = "application/json";
```

Comment:
Please look through Servlet API and Guava constants. I think they have this type defined already.